Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -181,7 +181,7 @@ function initialize(model) {
     .then(function (loader) {
       Matrix4.multiply(
         model._modelMatrix,
-        loader.transform,
+        loader.components.transform,
```

Comment:
Why does the `loader` need to persist again? Is it because it may still be loading textures etc. after the model has initialized? When implementing transcoding for I3DM, it'll need to pass this `components.transform` as a uniform, and it's not ideal to have to extract that from the loader (or maybe the scene graph since it also keeps the `components` as a property, however, it's not passed into any pipeline stage yet). Either way, I'd just like to double check that the loader does indeed need to persist and that we can't just hand off the components to the model.

Reply:
The loader needs to persist for incremental texture loading and so that the loader can be destroyed when the model is destroyed, for cache reference counting purposes. But it's also fine if loader hands off components to model since the loader shouldn't be accessed besides those two cases.